tests/unit/client-eviction.tcl

```diff
@@ -531,5 +531,87 @@ start_server {} {
     } {} {needs:debug}
 }
 
+start_server {} {
+    test "client total memory grows during client no-evict on" {
+        r client no-evict on
+
+        r setrange k [mb 1] v
+        set rr [redis_deferring_client]
+        set cname "test_client"
+        $rr client setname $cname
+        $rr client no-evict on
+        $rr flush
+
+        assert {[$rr read] == "OK"}
```

Comment:
aren't we missing the `read` of the `no-evict`?

btw we can create a normal client and then do `rr deferred 1` later.